install_pim/docker/installation_docker.rst

```diff
@@ -51,7 +51,7 @@ The following command will create a PIM project in the current directory. Please
     $ cd pim
     $ docker run -ti -u www-data --rm \
         -e COMPOSER_MEMORY_LIMIT=4G \
-        -v $(pwd):/srv/pim -v ~/.composer:/var/www/.composer -w /srv/pim \
+        -v $(pwd):/srv/pim -v ~/.composer:/var/www/.composer -w /srv/pim -v ~/.cache:/var/www/.cache \
```

Comment:
According to the composer documentation, the composer cache directory path can vary from one unix system to another so it won't work for anyone: https://getcomposer.org/doc/06-config.md#cache-dir.
Also, can there be an impact to mount all the `~/.cache` directory? There is a lot of cache in it, not only composer on, tt can be several GB.

Reply:
Maybe @BitOne you have a feedback on this?

Reply:
A mount is just a kind of link, so the size doesn't matter (nothing is copied or transferred).

According to the documentation, on Unix systems it's either .cache/composer or .composer/cache. As we mount both directories, we cover the two possibilities.